app/controllers/rents_controller.rb

```diff
@@ -3,11 +3,14 @@ class RentsController < ApiController
   before_action :set_locale
 
   def index
-    render_paginated Rent.all
+    rents = Rent.all
```

Comment:
Now that we have current_user to authenticate and know which is the current user, we can show the rents for that user. `Rent.where(user: current_user)`